client/src/Components/Layouts/ToBuildLater/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './style.css';
+import { Spin, Icon } from 'antd';
+
+
+
+export default function index() {
```

Comment:
@denaHS Here's another component where you could use a more descriptive name, because React devtools will show that name in its inspector.